base/src/commonTest/kotlin/utils/DecimalTest.kt

```diff
@@ -45,9 +44,19 @@ class DecimalTest {
         assertEquals(0.333, 1.toDecimal().div(3.toDecimal(), 3, RoundingMode.RoundDown).toDouble())
         assertEquals(0.334, 1.toDecimal().div(3.toDecimal(), 3, RoundingMode.RoundUp).toDouble())
 
-        assertFailsWith(DecimalException::class) { 1.0.toDecimal() / 0.0.toDecimal() }
-        assertFailsWith(DecimalException::class) { 1.0.toDecimal().div(0.0.toDecimal(), 3) }
-        assertFailsWith(DecimalException::class) { 1.0.toDecimal().div(0.0.toDecimal(), 3, RoundingMode.RoundDown) }
+        assertEquals(Decimal.PositiveInfinity, 1.0.toDecimal() / 0.0.toDecimal())
```

Comment:
Well, no more exceptions on division by zero?

Reply:
No, in consistency with Kotlin Double
https://pl.kotl.in/FQsIIQzI4